server/src/classes/ServerHub.ts

```diff
@@ -17,6 +18,7 @@ export default class ServerHub {
     private _ready = false;
 
     public constructor() {
+        this.app.use(cors());
```

Comment:
I would pass a corsOptions to prevent the default origin being set to '*', otherwise credentials will not send. For adding multiple origins (for localhost + 127.0.0.1 + any other host) see https://www.npmjs.com/package/cors#configuring-cors-w-dynamic-origin